test/test_examples_1d_burgers.jl

```diff
@@ -0,0 +1,19 @@
+module TestExamples1DBurgers
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "1d")
+
+@testset "Inviscid Burgers" begin
+  @testset "elixir_burgers_basic.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_burgers_basic.jl"),
+      l2   = [2.967470209082194e-5],
+      linf = [0.00016152468882624227])
+  end
+end
```

Comment:
Did you check EOC?

Reply:
Yes, N = 3 and N = 4 gives EOCs of 4 and 5 respectively.